go/cpufreqgetter/README.md

```diff
@@ -0,0 +1,6 @@
+# cpufreqgetter
+C and golang code to build minimal images to get read and dump data from /proc/cpuinfo
+
+**build image**
+
+docker build -t irfdocker/getcpufreq .
```

Comment:
Change this.  Work with Billy on integration with jenkins.

Reply:
I will work with Billy to build the multi-arch images after the code get merged in.

Reply:
Please create a Jira for this.